SCHNDR/iocBoot/iocSCHNDR-IOC-01/devices/GEMGateValue.cmd

```diff
@@ -0,0 +1,20 @@
+##
```

Comment:
Can we call this file `GEMGateValve.cmd` rather than `GEMGateValue.cmd`?

Reply:
not sure i can even blame autocorrect :-D yes will change